go-controller/pkg/ovn/master.go

```diff
@@ -797,42 +796,38 @@ func (oc *Controller) addNodeAnnotations(node *kapi.Node, hostSubnets []*net.IPN
 			v6Addr = ip
 		}
 	}
-	gwLPRAnnotations, err := util.CreateNodeGateRouterLRPAddrAnnotation(v4Addr, v6Addr)
-	if err != nil {
-		return fmt.Errorf("failed to marshal node %q annotation for Gateway LRP IP %v",
-			node.Name, gwLRPIPs)
-	}
 
-	nodeAnnotations, err := util.CreateNodeHostSubnetAnnotation(hostSubnets)
-	if err != nil {
-		return fmt.Errorf("failed to marshal node %q annotation for subnet %s",
-			node.Name, util.JoinIPNets(hostSubnets, ","))
-	}
+	//// Retry if it fails because of potential conflict which is transient. Return error in the
```

Comment:
only need // or use /*

Reply:
will fix.

Reply:
> Still not using the informer here to get the pod?

this has chaned to use oc.watchFacotory.GetPod()